### PR TITLE
Flex select pvr

### PR DESCRIPTION
**Pull request by darcyadams**
We switched changeup over to Avant Garde, which apparently has it's quirks. It was making select PVR look jacked up. 
Solved this by using a flex flex layout in the options instead of a absolute with inline styles calced in js.

### CODE REVIEW

File: src/SelectPvr.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default class SelectPvr extends React.Component<ISelectPvrProps, ISelectP
 
     this.state = {
       selectedOptions: defaultSelected,
-      aggregateOptionsHeight: (options.length * optionHeight) + (options.length * 1),
+      aggregateOptionsHeight: options.length * optionHeight,
```
Comment (darcyadams) [inline src/SelectPvr.tsx]: this was adding 5 extra pixels at the bottom of `selectPvr`s. I think it was meant to account for the borders, we must have switched  to using `border-box` at some point, making this code obsolete.

File: src/styles/pvr.styl [hunk 2]
```diff
@@ -164,30 +164,37 @@
...
       padding-left: 10px
       padding-right: 10px
       z-index: 0
 
-      &:hover
-        background-color: optionHoverLightBlue
-
       .label
+        font-size: 14px
+        height: 12px
```
Comment (darcyadams) [inline src/styles/pvr.styl]: for some reason Avant Garde is 2 pixels shorter than it's set size. It also pushes to the top of the container. So if I leave the height of the label container to `auto`, it is 14px tall, which screws up the vertical centering of the text by 2px. Only way I could solve this was to set the container to 2px smaller than the font-size. Open top other ideas. Tested this in chrome, FF, Desktop Safari and mobile.

File: views/index.pug [hunk 3]
```diff
@@ -5,6 +5,14 @@ html(lang='en')
...
     meta(name='viewport' content='width=device-width, initial-scale=1')
     title Lineup Lab
+    style(type="text/css").
+      @font-face {
+          font-family: 'ITCAvantGardeStd';
+          src: url('/styles/ITCAvantGardeStd-Bk.ttf') format('truetype');
+      }
+      body {
+        font-family: 'ITCAvantGardeStd';
+      }
```
Comment (darcyadams) [inline views/index.pug]: only wanted this font import to apply to lab, so put here to keep it out of any styles used consuming app.

File: src/styles/pvr.styl (hunk 2 above)
Comment (andyparisi) [reply src/styles/pvr.styl]: weird. i think that's fine for now.

File: src/styles/global.styl [hunk 4]
```diff
@@ -85,7 +83,6 @@ input, select, textarea
   
 
 textarea
-  font-family: "HelveticaNeue", Helvetica, Arial, sans-serif
```
Comment (andyparisi) [inline src/styles/global.styl]: I'm glad you removed these default fonts. I was meaning to do this, as it was screwing up the apps that imported lineup.

**Pull request merged**